app/plugin/grist-plugin-api.ts

```diff
@@ -594,18 +594,23 @@ onThemeChange((newTheme) => {
   attachCssThemeVars(_theme);
 });
 
-function attachCssThemeVars({appearance, colors}: any) {
+function attachCssThemeVars({appearance, colors: cssVars}: any) {
   // Prepare the custom properties needed for applying the theme.
-  const properties = Object.entries(colors)
+  const properties = Object.entries(cssVars)
     .map(([name, value]) => `--grist-theme-${name}: ${value};`);
 
   // Include properties for styling the scrollbar.
   properties.push(...getCssScrollbarProperties(appearance));
 
   // Apply the properties to the theme style element.
-  getOrCreateStyleElement('grist-theme').textContent = `:root {
+  // The 'grist-theme' layer takes precedence over the 'grist-base' layer where
```

Comment:
Does it matter here, in the plugin api? There is no grist-base layer here.

Reply:
I guess it does not right now, I wrapped the css rules in the layer to match how the theme vars are attached in the app. Felt better to have matching behavior. Will be easier to understand later if we want to add custom css also in the grist-plugin-api maybe?

Would you prefer I remove it?